Frontend/src/Components/Dashboard/Dashboard.js

```diff
@@ -313,7 +313,7 @@ export default function Dashboard(props) {
 
       <DataRecordingControl/>
 
-      <Grid margin={0.5} gap={1} flex="1 1 0" templateRows="2fr 3fr 3fr" templateColumns="1fr 1fr" >
+      <Grid margin={0.5} gap={1} flex="1 1 0" templateRows="3fr 3fr 3fr" templateColumns="1fr 1fr" >

```

Comment:
I haven't ran the code yet, but I have a feeling that this is going to cause data views to overflow. Would you be able to try to decrease the image size and rearrange the grid layout instead?

Reply:
I changed the `imageHeight` constant to 5 instead of the size of the container and everything appears to fit good on my end, can you check to make sure it fits in your testing too?
<img width="613" alt="image" src="https://github.com/badgerloop-software/chase-car-dashboard/assets/17338790/f11f3fcf-e728-4ec1-9e43-c2c41ebaecb7">

Reply:
I did some additional poking around, and I think it would be a good idea to make the SimpleGrid 3x7 instead of 4x6. Then, the grid will be fully populated, and there should be room to increase the vertical spacing

Reply:
<img width="460" alt="image" src="https://github.com/badgerloop-software/chase-car-dashboard/assets/17338790/825956ee-54d7-4999-93b8-9fb669200302">

I adjusted it to be a 3x7 grid and changed the spacing so that it looks pretty consistent on all sides, do you still want me to put the entire thing in a <Center> or use some CSS to try to center it further?

Reply:
Yeah, please do

Reply:
I set width and height to 95% and margins to auto, looks like it responds nicely to changing window sizes